apps/vr-tests-react-components/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@fluentui/vr-tests-react-components",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26312&commitId=99374d2dc00673b581f9f5b34c28ab29ab67f279&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1751 screenshots unchanged, 0 screenshots with different dimensions and 9 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.Subtle - Dark Mode.default.chromium.png|128| Changed|
|Button Converged.Subtle - Dark Mode.hover.chromium.png|147| Changed|
|Button Converged.Subtle - Dark Mode.pressed.chromium.png|155| Changed|
|Button Converged.Subtle - High Contrast.default.chromium.png|189| Changed|
|Button Converged.Subtle - High Contrast.hover.chromium.png|129| Changed|
|Button Converged.Subtle - High Contrast.pressed.chromium.png|129| Changed|
|Button Converged.Subtle.default.chromium.png|140| Changed|
|Button Converged.Subtle.hover.chromium.png|155| Changed|
|Button Converged.Subtle.pressed.chromium.png|139| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>